### PR TITLE
Add help in the FAQ for GPU usage

### LINKED ISSUE #271: [DOC] - Add GPU to the FAQ
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

When trying to run deep learning code on a GPU, the deep learning library does not always recognize and use the GPU, so the user should be be made aware that there is an easy solution.

### Steps to Resolve this Issue

1. Add to the FAQ to explain the solution.  It only takes a couple of sentences so this doesn't require a larger how-to or tutorial.
2.
3.
...

### PR DESCRIPTION
**Pull request by ericdatakelly**
## Reference Issues or PRs

Fixes #271 
Related: #119 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: docs/docs/faq.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ The `conda` config is located in the `/home/{user}/.condarc` file. You can chang
 
 You may find that the pods hosting your environment get full over time, prompting you to clear them out. To delete old builds of your environment on conda-store, click the "delete" button in the conda-store UI.
 
+## Why doesn't my code recognize the GPU(s) on Nebari?
+
+First be sure you chose [a GPU-enabled server when you selected a profile][selecting a profile].  Then be sure your environment includes a GPU-specific version of either PyTorch or TensorFlow, i.e. `pytorch-gpu` or `tensorflow-gpu`.  `tensorflow>=2` includes both CPU and GPU capabilities, but if the GPU is still not recognized by the library, try removing `tensorflow` from your environment and adding `tensorflow-gpu` instead.
```
Comment (kcpevey) [inline docs/docs/faq.md:81]:
```suggestion
First be sure you chose a [GPU-enabled server when you selected a profile][selecting a profile].  Next, be sure your environment includes a GPU-specific version of either PyTorch or TensorFlow, i.e. `pytorch-gpu` or `tensorflow-gpu`.  Also note that `tensorflow>=2` includes both CPU and GPU capabilities, but if the GPU is still not recognized by the library, try removing `tensorflow` from your environment and adding `tensorflow-gpu` instead.
```

**Review by kcpevey — APPROVED** [review]
Approve after changes. Looks good!

File: docs/docs/faq.md (hunk 1 above)
Comment (ericdatakelly) [reply docs/docs/faq.md:81]: Thanks!

**Comment by ericdatakelly** [comment]
This is ready to merge.  I don't have permission though.

**Pull request merged**